src/twisted/test/test_tcp.py

```diff
@@ -1363,15 +1363,21 @@ def test_buildProtocolClient(self):
 
 
 class LargeBufferWriterProtocol(protocol.Protocol):
-
     # Win32 sockets cannot handle single huge chunks of bytes.  Write one
     # massive string to make sure Twisted deals with this fact.
 
     def connectionMade(self):
         # write 60MB
```

Comment:
do we still write 60MB? it looks like there are 60MB-1 ... I guess that we can remove this comment

Reply:
commited change